src/components/sessions/session-detail-view/AttendanceTable.tsx

```diff
@@ -228,6 +237,26 @@ const AttendanceTable = ({
                 }
               />
             ),
+            customHeadLabelRender: () => (
+              <div style={{ width: 60, textAlign: "center" }}>
```

Comment:
the useStyles hook would make it easier to maintain, compared to inline styling

Reply:
yep! sorry sorry brittany left that comment too but i forgot to fix that along with the aria-labels. will take a look!